types/google-cloud__storage/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare namespace Storage {
         acl: Storage.Acl;
         combine(sources: string[] | File[], destination: string[] | File[]): Promise<[File, Storage.ApiResponse]>;
         create(config?: BucketConfig): Promise<[Bucket, Storage.ApiResponse]>;
+        createResumableUpload(options?: ResumableUploadOptions): Promise<[string]>;
```

Comment:
@seyfert I believe that this should be on the `class File` instead of on the Bucket class.
See available methods on here https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/0.56.0/storage/file and here https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/0.56.0/storage/bucket